core/controllers/editor_test.py

```diff
@@ -114,15 +111,15 @@ def test_editor_page(self):
         # Check that non-editors can access, but not edit, the editor page.
         response = self.get_html_response('/create/0')
         self.assertIn('Help others learn new things.', response.body)
-        self.assert_cannot_edit(response.body)
+        self.assert_cannot_edit(0)
```

Comment:
Shouldn't exploration IDs be strings?

Also, it's weird to have this here, since this line no longer has any direct relevance to the previous ones. There should be separate comprehensive tests for the new handler instead, as @DubeySandeep suggested. Ditto below.

Reply:
Done.